pyproject.toml

```diff
@@ -236,7 +236,7 @@ exclude = '''
 '''
 
 [tool.pytest.ini_options]
-addopts = ["-rfEX", "-pbentoml.testing.pytest.plugin"]
+addopts = ["-rfEX", "-pno:warnings"]
```

Comment:
Why are we doing this?

Reply:
Yes. Right now if users are running tests from our examples, it would be pretty strange if they already have a models for that example locally, then have to train that model again and run through the whole example tests.